### PR TITLE
EDL - increase timeout and docs

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-21890)

## Description
Increasing the NGINX Timeout to 3600 (was 1800), and improving the documentation incase of large number of indicators

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/README.md [hunk 1]
```diff
@@ -12,10 +12,15 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
...
 7. Follow the instructions in the rest of this guide to make sure that the PAN-OS device is connected to the EDL service.
 
-***Important Note:***
-EDL is designed to spawn on two processes: nginx and python. Nginx is the process that listens on the configured port, while the python process listens on the configured port + 1. This means that if an integration was configured for port 9009, the nginx process will listen on port 9009 and python on port 9010.
-When running without --network=host the python port is not exposed to the machine.
+***Important Notes:***
+- EDL is designed to spawn on two processes: nginx and python. Nginx is the process that listens on the configured port, while the python process listens on the configured port + 1. This means that if an integration was configured for port 9009, the nginx process will listen on port 9009 and python on port 9010. When running without --network=host the python port is not exposed to the machine.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/README.md:16]:
```suggestion
- EDL is designed to spawn on two processes: NGNIX and Python. NGNIX is the process that listens on the configured port, while the Python process listens on the configured port + 1. This means that if an integration was configured for port 9009, the NGNIX process will listen on port 9009 and Python on port 9010. When running without --network=host, the Python port is not exposed to the machine.
```

**Comment by ShirleyDenkberg** [comment]
@dantavori @DeanArbel Doc review completed.

**Review by kobymeir — APPROVED** [review]

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 2]
```diff
@@ -66,3 +66,7 @@ Use the following arguments in the URL to change the request:
 
 ### When Running in On-Demand Mode
 Make sure you run the ***!export-indicators-list-update*** command for the first time to initialize the export process.
+
+### Important Notes:
+- When constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended to instead use different instances of the EDL (one for each query), and set each one with a default query for better performance.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:71]:
```suggestion
- If constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended to use different instances of the EDL (one for each query), and set each one with a default query for better performance.
```

File: Packs/EDL/Integrations/EDL/README.md [hunk 3]
```diff
@@ -12,10 +12,16 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
...
 
-***Important Note:***
-EDL is designed to spawn on two processes: nginx and python. Nginx is the process that listens on the configured port, while the python process listens on the configured port + 1. This means that if an integration was configured for port 9009, the nginx process will listen on port 9009 and python on port 9010.
-When running without --network=host the python port is not exposed to the machine.
+***Important Notes:***
+- EDL is designed to spawn on two processes: NGNIX and Python. NGNIX is the process that listens on the configured port, while the Python process listens on the configured port + 1. This means that if an integration was configured for port 9009, the NGNIX process will listen on port 9009 and Python on port 9010. When running without --network=host, the Python port is not exposed to the machine.
+- When constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended instead to use different instances of the EDL (one for each query), and set each one with a default query for better performance.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/README.md:17]:
```suggestion
- If constantly using different queries for the same EDL instance through the *q* inline argument, it is recommended to use different instances of the EDL (one for each query), and set each one with a default query for better performance.
```

File: Packs/EDL/Integrations/EDL/README.md [hunk 4]
```diff
@@ -12,10 +12,16 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
...
+- When using the *q* inline argument, the number of exported indicators is limited to 100,000 due to performance reasons. To export more than 100,000 indicators, create a new instance of the integration with the desired Indicator Query and List Size.
 
+
+## Troubleshooting
+- If you are encountering an 504 Gateway error:
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/README.md:22]:
```suggestion
- If you are encountering a 504 Gateway error:
```

**Comment by ShirleyDenkberg** [comment]
@kobymeir @DeanArbel @anara123 @dantavori Doc review completed.

**Pull request merged**